tasks/install_targz.yml

```diff
@@ -24,5 +23,26 @@
     dest: "{{ nrinfragent_tarball_download_dir }}/newrelic_infra/linux_{{ nrinfragent_tarball_version }}_{{ nrinfragent_architecture }}/"
     remote_src: yes
     creates: "{{ nrinfragent_tarball_download_dir }}/newrelic_infra/linux_{{ nrinfragent_tarball_version }}_{{ nrinfragent_architecture }}/newrelic-infra/"
-  notify:
-    - running installer
+  register: archive_contents
+
+- name: running installer
+  command: ./installer.sh
+  args:
+    chdir: "/opt/newrelic_infra/linux_{{ nrinfragent_tarball_version }}_{{ nrinfragent_architecture }}/newrelic-infra/"
+  environment:
+    NRIA_BIN_DIR: "{{ nrinfragent_tarball_bin_dir | default('') }}"
+    NRIA_MODE: "{{ nrinfragent_tarball_mode | default('') }}"
+    NRIA_USER: "{{ nrinfragent_tarball_user | default('') }}"
+    NRIA_CONFIG_FILE: "{{ nrinfragent_tarball_config_file | default('') }}"
+
```

Comment:
is there a reason for the blank line?

Reply:
@cristianciutea can you take a look at this?